src/components/SearchBar.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+
+const GoogleBooksSearch = () => {
+  const [query, setQuery] = useState(""); // User input
+  const [books, setBooks] = useState([]); // Search results
+
+  // Function to handle the API request
+  const fetchBooks = async () => {
+    if (!query.trim()) return; // Avoid empty searches
+
+    try {
+      const response = await fetch(
+        `https://www.googleapis.com/books/v1/volumes?q=${encodeURIComponent(query)}`
+      );
+      const data = await response.json();
+
+      if (data.items) {
+        setBooks(data.items);
+      } else {
+        setBooks([]);
+      }
+    } catch (error) {
+      console.error("Error fetching books:", error);
+    }
+  };
+
+  // Function to handle form submission
+  const handleSubmit = (e) => {
+    e.preventDefault(); // Prevent page reload
+    fetchBooks();
+  };
+
+  return (
+    <div style={{ padding: "20px", fontFamily: "Arial, sans-serif" }}>

```

Comment:
According to the task you need to use `BEM` class names for styling the project. Please, don't use `style={{` (inline styles) in the code. It's hard to reuse and maintain

**Please, correct it everywhere in the code**

Reply:
[fix: delete extra file] - This file was a first attempt and I forgot to delete it as it wasn't how I ended up implementing this functionality